### PR TITLE
mynewt-mcumgr-cli: update to latest newt/newtmgr deps

### PR DESCRIPTION
**Pull request by nkaje**
Update the newt and newtmgr hashes to the latest.

Signed-off-by: Naveen Kaje <naveen.kaje@juul.com>

### CODE REVIEW

**Comment by nkaje** [comment]
Get fix to https://github.com/apache/mynewt-mcumgr-cli/issues/17

File: go.sum [hunk 1]
```diff
@@ -1,24 +1,36 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/JuulLabs-OSS/ble v0.0.0-20200716215611-d4fcc9d598bb h1:kIZ7fr8RxucJXNHifPxm71yiWuzpw0SmjlafMzoOd0U=
+github.com/JuulLabs-OSS/ble v0.0.0-20200716215611-d4fcc9d598bb/go.mod h1:6deIuswYSv6W1l3sM/nonw0OKWtIZCn7ZOWvIREoq2A=
+github.com/JuulLabs-OSS/cbgo v0.0.1/go.mod h1:L4YtGP+gnyD84w7+jN66ncspFRfOYB5aj9QSXaFHmBA=
 github.com/NickBall/go-aes-key-wrap v0.0.0-20170929221519-1c3aa3e4dfc5/go.mod h1:w5D10RxC0NmPYxmQ438CC1S07zaC1zpvuNW7s5sUk2Q=
+github.com/abiosoft/ishell v2.0.0+incompatible/go.mod h1:HQR9AqF2R3P4XXpMpI0NAzgHf/aS6+zVXRj14cVk9qg=
 github.com/abiosoft/readline v0.0.0-20180607040430-155bce2042db h1:CjPUSXOiYptLbTdr1RceuZgSFDQ7U15ITERUGrUORx8=
 github.com/abiosoft/readline v0.0.0-20180607040430-155bce2042db/go.mod h1:rB3B4rKii8V21ydCbIzH5hZiCQE7f5E9SzUb/ZZx530=
 github.com/apache/mynewt-artifact v0.0.3/go.mod h1:vFUd47t74KPQMzSBhQ2qp5Hc7D29OU/Tl3xHtFwN3k8=
```
Comment (utzig) [inline go.sum]: Any idea why this line was left here?

**Review by utzig — APPROVED** [review]
I always feel confused by these old package versions that are left on go.sum when updating, maybe we should get rid of them manually? Otherwise it looks OK.

**Comment by nkaje** [comment]
> I always feel confused by these old package versions that are left on go.sum when updating, maybe we should get rid of them manually? Otherwise it looks OK.

Thanks for the review. I will incorporate your comments and upload this again. I want to do one update that includes https://github.com/apache/mynewt-newtmgr/pull/177. So I will wait for that to review/merge.

**Comment by jensschroer** [comment]
Is it possible to merge this?

**Pull request merged**